release-team/role-handbooks/release-signal/README.md

```diff
@@ -0,0 +1,562 @@
+
+- 1/2 hour to 2 hours daily, every workday, checking and following up on failing/flaky tests, issues and PRs targeted for the
+  ongoing release cycle around week 6 (mid-release).
+- Availability to attend most of Release Team (weekly) and Burndown meetings (daily during Code Freeze), subject to time zone
+  appropriateness.
+- Ability to follow up on test fixes during Code Freeze at arbitrary times to ensure rapid turnaround.
+- The time commitment increases through the release cycle, peaking during Code Freeze. In the last week of Code Freeze, shadows
+  should expect to spend at least 10 hours and lead at least 20 hours per week.
+
+### Additional Requirements for Shadows
+
+The following requirements are all things that the lead should already have, and shadows should acquire, with coaching, within the
+first weeks of the release cycle:
+
+- Have signed the contributor CLA for Kubernetes.
+- Become a Kubernetes org member. In many cases, this will be done with the sponsorship of the Release Signal Lead or Release Lead
+  in the first week of the cycle.
+    - The process to become an org member is
+      in [our community membership ladder](https://github.com/kubernetes/community/blob/master/community-membership.md#requirements-for-outside-collaborators)
+- General familiarity with our [test boards](https://testgrid.k8s.io/) and examining test results from automated tests.
+- Willingness and ability to follow up with contributors about test failures on Slack, email, Discuss, and SIG meetings, as
+  appropriate.
+- Ability to file well-crafted Kubernetes issues, including labelling.
+- General familiarity with GitHub labels and how to find issues/PRs for the current milestone.
+- General knowledge of our SIG system and SIGs' areas of responsibility.
+
+Additionally, the following qualifications make a candidate more suitable for the Release Signal team but are not requirements:
+
+- Prior involvement with SIG Testing and the Test Infrastructure team.
+- Experience with automated testing, CI/CD, quality engineering, and/or quality assurance.
+- Experience with quality assurance and bug-tracking systems.
+- Experience with the GitHub automation.
+
+### Additional Requirements for Leads
+
+In addition to the above requirements for shadows, most of which become prerequisites, Release Signal Leads must:
+
+- Have the ability to add a milestone to issues, so must be a member of
+  the [milestone maintainers](https://github.com/orgs/kubernetes/teams/milestone-maintainers).
+- Have a working knowledge of our various test infrastructure tools, such as Testgrid, Triage, Spyglass, Prow, and Tide.
+- Have a working knowledge of GitHub labelling and labels used by the project. Release Signal leads must be able to identify
+  relevant issues and PRs for a given milestone.
+- Understand what defines a Release Blocking issue or PR, or know who to contact to determine that information.
```

Comment:
do we have a some doc about this?

Reply:
maybe in the release lead handbook

Reply:
Are you referring to the Release Signal Lead Checklist? I don't understand.